cmd/notice-file-generator/dependency.go

```diff
@@ -48,6 +48,8 @@ var regexpGoImport = []*regexp.Regexp{
 	regexp.MustCompile(`(?i)<\s*meta\s*content\s*=\s*"(?P<import_prefix>\S+)\s+(?P<vcs>\S+)\s+(?P<repo_root>\S+)"\s*name\s*=\s*"go-import"\s*/?>`),
 }
 
+var regexPythonRepo = regexp.MustCompile(`^#\s*[R|r]epo(sitory)*:\s*(?P<url>https:\/\/github.com\/(?P<full_name>.*\/(?P<name>.*)))`)
```

Comment:
Regex to support many formats in the pipfile and have whitespace handling. Also it will map subpaths to the variables. We only need those three to build dependency(url, full_name, name) .

To test regex: https://regex101.com/r/FKMaBO/1